Lets Do This/Controllers/Login/LDTUserRegisterViewController.m

```diff
@@ -181,6 +182,10 @@ - (IBAction)submitButtonTouchUpInside:(id)sender {
             }];
 
         } failure:^(NSError *error) {
+            // Only record error in Crashlytics if error is NOT lack of connectivity, timeout, or email already exists.
+            if ((error.code != -1009) && (error.code != -1001) && (error.code != 422)) {
```

Comment:
This check wasn't working -- #938. The 422 code we're looking for is nested within the response.